package_manager/dpkg_parser.py

```diff
@@ -45,6 +46,8 @@
                     help='A comma delimited list of packages to search for and download')
 parser.add_argument("--workspace-name", action='store',
                     help='The name of the current bazel workspace')
+parser.add_argument("--versionsfile", action='store',
+                    help='The target path of the versions file')
```

Comment:
"If set, the output path of the versions file to be generated"?

Reply:
Fixed